src/code/reducers/index.js

```diff
@@ -13,20 +13,22 @@ import gametes from './gametes';
 import drakes from './drakes';
 import baskets from './baskets';
 
-const initialState = Immutable({
-  template: null,
-  hiddenAlleles: ['t','tk','h','c','a','b','d','bog','rh'],
-  trial: 0,
-  case: 0,
-  challenge: 0,
-  challenges: 1,
-  challengeProgress: {},
-  authoring: window.GV2Authoring,
-  endCaseUrl: urlParams.start
-});
+function initialState() {
```

Comment:
After converting `gv2.js` to `gv2.json` I ran into order-of-initialization issues between `initialState` and `window.GV2Authoring`. Rather than figuring out how to control the initialization order it seemed simplest to remove `initialState` from static initialization time.